api/admin/controller.py

```diff
@@ -489,9 +650,11 @@ def edit(self, identifier_type, identifier):
                 classify=True,
                 regenerate_opds_entries=True,
                 update_search_index=True,
-                choose_summary=True
+                calculate_quality=True,
```

Comment:
I don't know how much time it will save to change this, but it occurs to me that you could pass in calculate_quality=True or choose_summary=True only if those fields changed in the current request. Those methods are potentially expensive, especially choose_summary, which performs textual analysis.